src/mudlet.cpp

```diff
@@ -3431,8 +3431,13 @@ void mudlet::show_options_dialog(QString tab)
         connect(mpActionReconnect.data(), &QAction::triggered, mpProfilePreferencesDlgMap.value(pHost)->need_reconnect_for_specialoption, &QWidget::hide);
         connect(dactionReconnect, &QAction::triggered, mpProfilePreferencesDlgMap.value(pHost)->need_reconnect_for_specialoption, &QWidget::hide);
         mpProfilePreferencesDlgMap.value(pHost)->setAttribute(Qt::WA_DeleteOnClose);
-        mpProfilePreferencesDlgMap.value(pHost)->setStyleSheet(pHost->mProfileStyleSheet);
     }
+
+    // pHost can be a nullptr here so we do not want to dereference it in that
+    // case {when there is no profile loaded}:
+    if (pHost) {
+        mpProfilePreferencesDlgMap.value(pHost)->setStyleSheet(pHost->mProfileStyleSheet);
+    }        mpProfilePreferencesDlgMap.value(pHost)->setStyleSheet(pHost->mProfileStyleSheet);
```

Comment:
Are you sure the closing brace is in the right spot?

Reply:
WTF! I do not remember that extra stuff after the `}` on new line 3440!

As for the remaining three lines in the method - yeah - they are still needed to return the `dlgProfilePreferences` associated with the `Host` pointer to the foreground - *even if it is a nullptr* because  there isn't a profile (yet) to associated the preferences with. There may be a corner case in which the user loads a profile whilst the dialogue is open - in which case the former should quietly slip it's details into all the relevant spots in the latter - but it might not acquire the styling that it would otherwise have if the preferences were opened after the profile was loaded.

BTW the primary reason behind allowing the preferences to be opened before a profile is: so the user can set their language the very first time Mudlet is started up - but it seems like I never to arranging for that in the block of code in the `ìf (first_launch)` part around 478 in `main.cpp`...